cekit/generator/base.py

```diff
@@ -294,6 +294,7 @@ def render_dockerfile(self):
         env.globals['helper'] = TemplateHelper(self._module_registry)
         env.globals['image'] = self.image
         env.globals['builders'] = self.builder_images
+        env.globals['cachito'] = self.image.get('osbs', {}).get('configuration', {}).get('container', {}).get('remote_source', {})
```

Comment:
I would like to remove this from here and access cachito configuration via Jinja template.